.github/workflows/linting.yml

```diff
@@ -1,29 +1,33 @@
-name: "Linting"
-
+name: WPCS check
```

Comment:
I think we can use a more descriptive name like WordPress Coding Standards or WordPress Linting or something like that. If you change that name, it may be a good idea to switch to a file name related to our workflow.

Reply:
Done.